### ISSUE TITLE
Updating owns email address

### ISSUE THREAD

**Issue by Matrix89** [bluesky-social/atproto#1127]
**Is your feature request related to a problem? Please describe.**

Yep, I've made a typo in my email address

**Describe the solution you'd like**

There should be a way to update the email address for non admin users.

I think for a MVP something similar to [`updateHandle`](https://github.com/bluesky-social/atproto/blob/main/packages/pds/src/api/com/atproto/identity/updateHandle.ts) would work.
In the future email verification should also be added

**Describe alternatives you've considered**

\-

**Additional context**

I know there was [this](https://github.com/bluesky-social/social-app/issues/413) issue on the frontend repo, but it didn't give me much insight, I hope APP-442 isn't in some deep backlog :grimacing:

If my proposed solution is ok, I'd like to work on a MVP, just give me a sign-off

**Comment by bnewbold** [comment]
This is a known and relatively urgent issue, and we want to incorporate email verification at the same time. It's not clear if we would merge an external contribution on this one vs. going with our own implementation, but I could be wrong.

**Comment by Matrix89** [comment]
Yeah, I've figured it might require some internal back and forth. I might shoot my shot any way :shrug:
